File: common/src/main/java/de/hype/bingonet/shared/packets/addonpacket/PublicChatAddonPacket.java
package de.hype.bingonet.shared.packets.addonpacket;

import de.hype.bingonet.environment.addonpacketconfig.AbstractAddonPacket;

/**
 * Sends the specified message to the server.
 */
public class PublicChatAddonPacket extends AbstractAddonPacket {
    public final String message;
    public final double timing;

    public PublicChatAddonPacket(String message, double timing) {
        super(1, 1); //Min and Max supported Version
        this.message = message;
        this.timing = timing;
    }
}

File: common/src/main/kotlin/de/hype/bingonet/shared/packets/addonpacket/PublicChatAddonPacket.kt
package de.hype.bingonet.shared.packets.addonpacket

import de.hype.bingonet.environment.addonpacketconfig.AbstractAddonPacket

/**
 * Sends the specified message to the server.
 */
class PublicChatAddonPacket(val message: String, val timing: Double) : AbstractAddonPacket(1, 1)
